build.gradle

```diff
@@ -64,6 +65,10 @@ configurations.all {
     resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
 }
 
+configurations.runtimeClasspath.resolutionStrategy.dependencySubstitution {
+    substitute module('org.jooq:jooq') with module("org.jooq:jooq:$jooq_version")
+}
```

Comment:
what is this for?

Reply:
Russell and I were getting this jooq error when running the gui fat jar, and he was able to fix it with that.

`Caused by: java.lang.VerifyError: class jooq.tables.AudioPluginEntity overrides final method org.jooq.impl.AbstractTable.fieldsRow()Lorg/jooq/Row;`

Per Russell:
> I actually fixed the jooq one. There was another error hiding right behind it though. My fix for the first one was to force a version on the jooq dependency.

Reply:
For clarity's sake, the error hiding behind it was the platform dependent javafx stuff that is also fixed in this PR.